src/components/checkBoxSelection/checkBoxSelection.scss

```diff
@@ -36,17 +28,62 @@
   margin-top: 20px;
   display: none;
 }
-
 .card-settings {
-  height: 600px;
+  height: 350px;
+  margin-left: 159px;
```

Comment:
this would make your page unresponsive

Reply:
margin-left: 159px; have been removed

Reply:
did you manage to fix the scrolling page?

Reply:
Yes, its no longer scrolling.

Reply:
great im glad you managed to find a solution